Pipfile

```diff
@@ -16,7 +16,6 @@ bs4 = "==0.0.1"
 colorlog = "==4.1.0"
 defusedxml = "==0.6.0"
 docker = "==4.2.0"
-fetch-p2p-api = {index = "https://test.pypi.org/simple/",version = "==0.0.2"}
```

Comment:
This has a dependency on fetchai api, since we don't need either; I am removing it